_cucumber/features/step_definitions/community.steps.rb

```diff
@@ -0,0 +1,19 @@
+Then(/^I should see the community page title$/) do
+  expect(@site.community.community_title).to be_truthy
+end
+
+Then(/^I should see at least "([^"]*)" promoted links$/) do |links|
+  expect(@site.community.promoted_image_links(links)).to be_truthy
+end
+
+Then(/^I should see at least "([^"]*)" community links$/) do |links|
+  @site.community.clear_filters
+  expect(@site.community.community_links(links)).to be_truthy
```

Comment:
OK point taken. I can change it to give a better error when failing though. Something like 

raise "Community title was not found" unless @site.community.community_title.eql?(true)